### ISSUE TITLE
cas/ldapadmin - allow to disable account creation via /account/new

### ISSUE THREAD

**Issue by landryb** [georchestra/georchestra#1047]
In some cases account creation is handled somewhere else, it would be nice to be able to disable totally account creation via cas. Removing the link to /account/new in casLoginView.jsp is easy, but disabling /account/new service/url from ldapadmin is not straightforward. Maybe a new boolean in the config, (along moderatedSignup) disabling it in https://github.com/georchestra/georchestra/blob/15.06/ldapadmin/src/main/java/org/georchestra/ldapadmin/ws/newaccount/NewAccountFormController.java#L85 ?

**Comment by fvanderbiest** [comment]
I agree with the necessity of a new parameter, but it should be a shared one, since it applies to CAS & ldapadmin at least.

**Comment by fvanderbiest** [comment]
No easy/clean way to do ATM to my knowledge.

Right now, best way is probably to:
 - prevent access to /account/new (with https://github.com/georchestra/georchestra/blob/15.06/security-proxy/src/main/webapp/WEB-INF/applicationContext-security.xml#L13-L42)
 - remove the links to ldapadmin from https://github.com/georchestra/template/blob/15.06/cas-server-webapp/WEB-INF/view/jsp/default/ui/casLoginView.jsp#L64-L66 in your own config
